src/commons/actor.rs

```diff
@@ -207,15 +205,15 @@ impl Actor {
         A: ToPolar + Display + Clone,
         R: ToPolar + Display + Clone,
     {
-        if let Some(error_msg) = &self.auth_error {
+        if let Some(api_error) = &self.auth_error {
             trace!(
-                "Unable to check access: actor={}, action={}, resource={}: {}",
+                "Authentication denied: actor={}, action={}, resource={}: {}",
```

Comment:
Looking at this again I think both the original log message and the new log message are incorrect.

In the original log message saying that we were unable to check access leaves it unclear (from that log statement alone) whether access is being granted or denied.

With the new log message it is also not clear, and it's also not true to say that authentication is/was denied. Instead what's happened is that the user was able to authenticate against the OpenID Connect 3rd party system (i.e. login to company X) but we were unable to complete the login to Krill itself perhaps due to missing information in the backend provider metadata about the user or a problem with connection to the backend after the user was authenticated by the 3rd party system but before Krill finished its part of the login process.

Any return from this function leads to one of two possible outcomes for the end user: access denied or access granted. I suppose you could also try to handle a 3rd scenario whereby a retry of a failure could be done, but you'd have to understand the root cause for why you didn't complete the login process properly which can't be known from here.

A nice simplification of this function would be to have only one log statement that mentions the affected actor, action being requested and resource to be affected, a statement of access granted or access denied, and in the denied case a reason why.

Reply:
Deferred to #420.